server/configs/application.properties

```diff
@@ -61,7 +61,7 @@ context.encryptionKey=@@encryptionKey@@
 
 #context.oldEncryptionKey=
 #context.requiredModules=
-#context.pipelineConfig=/path/to/pipeline/config/dir
+context.pipelineConfig=@@pathToServer@@/build/deploy/embedded/config
```

Comment:
Having a value for this property will cause the server to always use the "enterprise" pipeline (even if there's no JMS queue configured). I ultimately want to eliminate the non-enterprise implementation completely, but I'm not sure we're ready quite yet.

Can we easily make setting this property more conditional? If not, we can try this approach but will need to be looking more widely for pipeline problems, failure to shut down cleanly, etc.

Reply:
Sure. I can just have the task in sequenceAnalysis add this property, similar to how the premium module adds properties for ldapSync.